source/Octopus.Tentacle.Contracts/Logging/ITaskLog.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Octopus.Tentacle.Contracts.Logging
+{
+    public interface ITaskLog
```

Comment:
From tentacles perspective this is not necessarily a task, this should have a different name.

Reply:
Per the slack thread, what _is_ it if not the task? There wasn't an obvious name :-(

Reply:
This client is also used outside of octopus e.g. when configuring a DW.

What is it? A thing the client will log to.

Reply:
`ITentacleClientLogger` perhaps?

Reply:
As I think about it more, `ITaskLog` is the same name as an interface use internally in octopus, so by having one here it does feel like we are going to offer the chance of importing the wrong one or this interface being used in places it should not be.

A name like `ITentacleClientLogger` or even `ITentacleClientTaskLog` should prevent it from ever being used in the wrong place, which as I think about it is my larger concern.